packages/esm-patient-appointments-app/src/appointments/appointments-form.component.tsx

```diff
@@ -14,21 +14,21 @@ import {
   TimePicker,
 } from 'carbon-components-react';
 import { amPm, convertTime12to24 } from '@openmrs/esm-patient-common-lib';
-import { useLocations, useSessionUser, showToast, showNotification } from '@openmrs/esm-framework';
+import { useLocations, useSessionUser, showToast, showNotification, useLayoutType } from '@openmrs/esm-framework';
 import { appointmentsSearchUrl, createAppointment, useAppointmentService } from './appointments.resource';
 import styles from './appointments-form.scss';
 import { AppointmentPayload } from '../types';
 
 interface AppointmentsFormProps {
   closeWorkspace: () => {};
-  isTablet: boolean;
```

Comment:
Actually, would you mind deleting these custom interfaces and using `DefaultWorkspaceProps` when applicable?

Reply:
Ah right. I'll do it and update the PR

Reply:
Updated with f3582be

Reply:
Updated with f3582be